builder/Dockerfile

```diff
@@ -12,14 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.11.0
+FROM golang:1.16
 LABEL maintainer="Andy Xie <andy.xning@gmail.com>"
```

Comment:
@Random-Liu Should we have a separate PR to basically change this (and friends) to Kubernetes Authors?